sql/step_78_adjust_lab_datetime.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE OH_LABORATORY SET LAB_CREATED_DATE = LAB_DATE;
+UPDATE OH_LABORATORY SET LAB_DATE = CAST(CONCAT(LAB_EXAM_DATE, ' ', TIME(LAB_DATE)) AS DATETIME);
+
+ALTER TABLE OH_LABORATORY
+DROP COLUMN LAB_EXAM_DATE;
```

Comment:
I would put it in a singole line
and I would rename the file in `step_78_drop_lab_exam_date.sql` (like step_71)

Reply:
done; BTW, I just used the name that was there in the extra directory and changed XX to 78

Reply:
ok, thanks!